tools/build_litecore.bat

```diff
@@ -43,7 +43,7 @@ if not exist "%outputDir%" (
 
 rem Untested
 if "%lib%" == "LiteCore" (
-    "C:\Program Files\CMake\bin\cmake.exe" -G %VS_GEN% -DBUILD_ENTERPRISE=%entBuild% -DCMAKE_BUILD_TYPE=RelWithDebInfo ..\.. || goto :error
+    "C:\Program Files\CMake\bin\cmake.exe" -G %VS_GEN% -DENABLE_TESTING=OFF -DBUILD_ENTERPRISE=%entBuild% -DCMAKE_BUILD_TYPE=RelWithDebInfo ..\.. || goto :error
```

Comment:
I believe this is not strictly needed since I added this to be forced off in the LiteCore CMakeLists.txt but having it here hurts nothing.  Just an informational notice.  Ditto for the other non-mbedcrypto invocations.